SDF vs YANG (#8)
one-data-model/ietf108: charter.md

  ecosystems in a common way; they look towards the IETF for
  developing SDF 1.0 further into a high-quality specification.

  The IETF also defines the YANG data modelling language (RFC 7950).
  YANG has a strong focus on modelling the management interface for network devices,
  using a fairly small set of network management protocols,
  whereas SDF is designed to span (and transitively unify) many existing interaction models already present.
  While YANG can be serialized to a variety of wire formats (XML, JSON, CBOR, ...),

>   the serialization is driven directly by the structure of the YANG.

Reviewer: ; there is no separate "protocol binding" step as in the use of SDF.

Author: I don't know how to process this. Is it a suggestion?

Reviewer: Well, you just add the text after "the structure of the YANG" (or what you now made of that).

>   the serialization is driven directly by the structure of the YANG.

Author: ```suggestion
  the serialization is driven directly by the structure of the YANG; there is no separate "protocol binding" step as in the use of SDF.
```

Author: Is this what you had in mind?

>   While YANG can be serialized to a variety of wire formats (XML, JSON, CBOR, ...),
>   the serialization is driven directly by the structure of the YANG.

Author: ```suggestion
    While data associated with a YANG schema can be serialized to a variety
   of wire formats (e.g., XML, JSON, CBOR, and others), the serialization is
   driven directly by the structure of the data models defined in YANG; there is no separate "protocol binding" step as in the use of SDF.
```

>   While YANG can be serialized to a variety of wire formats (XML, JSON, CBOR, ...),
>   the serialization is driven directly by the structure of the YANG; there is no separate "protocol binding" step as in the use of SDF.

Author: ```suggestion
    While data associated with a YANG schema can be serialized to a variety
   of wire formats (e.g., XML, JSON, CBOR, and others), the serialization is
   driven directly by the structure of the data models defined in YANG; there is no separate "protocol binding" step as in the use of SDF.
```

  YANG also has an opinionated way in which the RPC is done.
  This is necessary in order to get the kind of cross-vendor interoperability needed for efforts in the Software Defined Networking.

>   On the other hand, SDF does not deal directly with serialization, dealing only in semantics of content, and leaving the existing IoT SDOs to do their own serialization.

Reviewer: I think I can live with s/semantics/structure and semantics/, as was suggested.
Also, s/content/data being interchanged/

>   On the other hand, SDF does not deal directly with serialization, dealing only in semantics of content, and leaving the existing IoT SDOs to do their own serialization.

Author: ```suggestion
  On the other hand, SDF does not deal directly with serialization, dealing only in structure and semantics of data being interchanged, and leaving the existing IoT SDOs to do their own serialization.
```

>   On the other hand, SDF does not deal directly with serialization, dealing only in semantics of content, and leaving the existing IoT SDOs to do their own serialization.

Author: ```suggestion
   Conversely, SDF does not deal directly with serialization at all,
   modelling only the structure and semantics of the data being interchanged, hence leaving data
   serialization (and RPC semantics) to other standards, most likely defined
   by existing IoT SDOs.

```

Reviewer: I cannot apply this, but it seems to be a good change.  Wow, github can be very confusing...

>   On the other hand, SDF does not deal directly with serialization, dealing only in structure and semantics of data being interchanged, and leaving the existing IoT SDOs to do their own serialization.
> 

Author: ```suggestion
   Conversely, SDF does not deal directly with serialization at all,
   modelling only the structure and semantics of the data being interchanged, hence leaving data
   serialization (and RPC semantics) to other standards, most likely defined
   by existing IoT SDOs.

```

  ecosystems in a common way; they look towards the IETF for
  developing SDF 1.0 further into a high-quality specification.

  The IETF also defines the YANG data modelling language (RFC 7950).
  YANG has a strong focus on modelling the management interface for network devices,
  using a fairly small set of network management protocols,
  whereas SDF is designed to span (and transitively unify) many existing interaction models already present.
    While data associated with a YANG schema can be serialized to a variety
   of wire formats (e.g., XML, JSON, CBOR, and others), the serialization is
   driven directly by the structure of the data models defined in YANG; there is no separate "protocol binding" step as in the use of SDF.

>   YANG also has an opinionated way in which the RPC is done.
>   This is necessary in order to get the kind of cross-vendor interoperability needed for efforts in the Software Defined Networking.

Author: ```suggestion
```
